test_assets/features/support/world.js

```diff
@@ -1,11 +1,10 @@
 module.exports = function() {
-  this.World = function(callback) {
-    var world = {};
+  this.World = function() {
```

Comment:
@simondean 
This change was introduced in cucmber-js 0.8.0, that is why I updated this part. Basically the callback is not used anymore in cucmber-js. But I can update to 
```
var world = {};
...
return world;
```
if you want. It both works.